### PR TITLE
Style guide tweaks

### PR DESCRIPTION
**Pull request by austin-schaefer**
@paperclypse : A collection of minor tweaks to style guide, along with fixes for some nav issues.

@DEVEX: Please hold off on merging this one, I'd like a tech writer reviewer.

### CODE REVIEW

File: src/content/docs/style-guide/processes-procedures/rename-or-redirect-document.mdx [hunk 1]
```diff
@@ -18,12 +18,12 @@ This document describes how to change the title of a document and how to create,
 
 ## URL format [#title-url]
 
-Changing a document's title also affects its URL. The URL is the text of the title, where hyphens `-` replace spaces. Drupal automatically removes ["short words"](/admin/config/search/path/settings) (`the`, `a`, `for`, `via`, etc.) from URL paths.
+Changing a document's title also affects its URL. The URL is the text of the title, where hyphens `-` replace spaces.
```
Comment (jerelmiller) [inline src/content/docs/style-guide/processes-procedures/rename-or-redirect-document.mdx:21]: This is no longer true in the new Gatsby site. The URL is determined by the filepath in `src/content`. So a file at `src/content/docs/test.mdx` will create a page at `/docs/test`. The `title` is now just to show on the top of the page, and will render as part of the title in the tab of the browser.

File: src/content/docs/style-guide/processes-procedures/use-content-types-text-formats.mdx [hunk 2]
```diff
@@ -60,7 +60,7 @@ Thr top of every doc begins with a set of metadata. Read on for more information
 
     <tr>
       <td>topics</td>
-      <td>Keywords related to your doc. Start each topic with a ** _ ** on a new line. 
+      <td>Keywords related to your doc. Start each topic with a `_`* on a new line.
```
Comment (jerelmiller) [inline src/content/docs/style-guide/processes-procedures/use-content-types-text-formats.mdx:63]: We've renamed `topics` to `tags`. Should that be captured here?

File: src/content/docs/style-guide/quick-reference/clamshells.mdx [hunk 3]
```diff
@@ -120,5 +120,6 @@ Here's an example of the clamshell source:
 To open or close a clamshell:
 
 * Click the open buttons or **Show/Hide All**.
-* Arrive at an individual clamshell via its URL. For example, go directly to [Clamshell 1 in the example above](/docs/new-relic-only/basic-style-guide/writing-guidelines/clamshells#clamshell-1).
+* Arrive at an individual clamshell via an anchor ID. For example, go directly to [Clamshell 1 in the example above](/docs/new-relic-only/basic-style-guide/writing-guidelines/clamshells#clamshell-1).
```
Comment (jerelmiller) [inline src/content/docs/style-guide/quick-reference/clamshells.mdx:123]: [nit] We've renamed "clamshell" to "collapser". Should we use that language in here?

File: src/content/docs/style-guide/writing-guidelines/levels-headings.mdx [hunk 4]
```diff
@@ -35,11 +35,11 @@ If your doc's title is long and you would like a shorter title to appear in the
 Organize chunks of information into sections with level two headings (`##`). For example:
 
 ```
-## Create a new user
+## Create a new user  [#create-new-user]
 ```
 
 <Callout variant="important">
-  Our site will use your header text as that header's ID.
+  If you don't specify an ID manually, the site will use your header text as that header's ID. Creating a manual ID can help preserve anchor links when docs move around and get rewritten.
```
Comment (jerelmiller) [inline src/content/docs/style-guide/writing-guidelines/levels-headings.mdx:42]: [nit] The manual `ID` (or custom`ID`, whichever term you like 😜 ), isn't so much about preserving the location when a doc moves, but rather if the title changes. Without the custom ID, the site will create a derived ID by using the text in the heading. So for example "Ruby agent examples", would get an automatically generated ID of `ruby-agent-examples`. If this heading were to change to `The world's best ruby agent examples`, then the auto ID would get generated as `the-worlds-best-ruby-agent-examples`, and you'd lose reference to the old one.

Moving a page doesn't have any affect on the ID itself.

**Review by jerelmiller — COMMENTED** [review]
Figured I'd add a bit of engineering related feedback

File: src/content/docs/style-guide/processes-procedures/rename-or-redirect-document.mdx (hunk 1 above)
Comment (paperclypse) [reply src/content/docs/style-guide/processes-procedures/rename-or-redirect-document.mdx]: I've rewritten to reflect the reality here.

File: src/content/docs/style-guide/processes-procedures/use-content-types-text-formats.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/style-guide/processes-procedures/use-content-types-text-formats.mdx:63]: Yup! I've updated this doc.

File: src/content/docs/style-guide/quick-reference/clamshells.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/style-guide/quick-reference/clamshells.mdx]: Yup, I've updated that, thanks!

**Review by paperclypse — APPROVED** [review]
Looks good to me! I made some minor edits based on @jerelmiller's suggestions.

**Comment by paperclypse** [comment]
I fixed those merge conflicts, @austin-schaefer

**Pull request merged**